package.json

```
@@ -10,14 +10,16 @@
     "build:hot": "tsc -w --preserveWatchOutput",
     "build": "tsc -b",
     "clean": "rm -rf dist/",
-    "dev": "",
+    "dev": "node -r dotenv/config --async-stack-traces --max-old-space-size=920 dist/index.js",
     "doctoc": "doctoc --notitle --maxlevel 3 README.md",
     "lint": "eslint --ext=jsx,ts,tsx,js .",
     "prepare": "husky install",
-    "start": "UV_THREADPOOL_SIZE=64 node -r dotenv/config --async-stack-traces --max-old-space-size=920 dist/index.js",
+    "start": "UV_THREADPOOL_SIZE=64 node --async-stack-traces --max-old-space-size=920 dist/index.js",
     "start_new": "indexName=npm-search-new bootstrapIndexName=npm-search-new.tmp UV_THREADPOOL_SIZE=64 node --async-stack-traces --max-old-space-size=920 dist/index.js",
     "test:watch": "jest --watchAll --no-watchman",
-    "test": "jest --forceExit"
+    "test": "jest --forceExit",
+    "docker:build": "./scripts/build.sh",
+    "docker:release": "VERSION=$(npx json -f package.json version) && docker push algolia/npm-search && docker push algolia/npm-search:$VERSION"
```

don't we want json to be a dependency if used in a script?

---

it could, we usually don't do it because it's not a requirement to run and dev on the project

---

```suggestion
    "docker:release": "VERSION=$(node -e 'console.log(require(\'./package.json\').version)') && docker push algolia/npm-search && docker push algolia/npm-search:$VERSION"
```

What do you think of this version instead? That way nothing can go wrong with the dependency

---

allez